examples/shallow/execute.py

```diff
@@ -94,7 +94,7 @@ def execute(option):
                                     stdout=subprocess.PIPE)
     if make_process.wait() != 0:
         return False, []
-    _ = make_process.stderr.read()
+    # _ = make_process.stderr.read()
```

Comment:
Thanks for the tip, I'll remember that. Yes it was because of it not accepting "_" as an unused variable. To be fair, I only added it to show people how to access stderr so it is just as valid being a comment in my opinion.